.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+name: test on CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [2.7, 3.0, 3.1, 3.2, head]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - name: Run tests
+        run: bundle exec rake test:units lint
```

We need to move `lint` out into its own job. The version of Rubocop we are using does not work on newer versions of Ruby, so we can't have it run as mart of the Ruby matrix.

```suggestion
        run: bundle exec rake test:units
```

A separate job for Rubocop could look like this:

```yaml
  rubocop:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: "2.7"
          bundler-cache: true
      - name: Run rubocop
        run: bundle exec rake lint
```

What do you think?

---

IMO, It is better to update Rubocop version and settings propery, but It is not appropriate on this PR.
Another PR is proper.

Separating a test and a lint is sounds good for me, 1 task 1 purpose.
So I separate test and lint on this PR, thank you example one :pray:
Updating settings or so will try by future PR by someone.